### ISSUE TITLE
As a site editor, I can no longer see the confusing grid options available under the "Styles" menu within each layout section

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#5351]
# Story

As a site editor, I can no longer see the confusing grid options available under the "Styles" menu within each layout section.

The current grid layout options appear under the `Styles` menu for all layouts.   They are confusing and often misused because they only work on one column layouts with certain components that were meant for uiowa.edu.  

<img width="323" alt="Screen Shot 2022-06-08 at 2 14 10 PM" src="https://user-images.githubusercontent.com/1036433/172698133-adafd73f-d646-4eea-92c2-5516d8eb8f1a.png">

# Proposed solution

- Remove the grid styles `Layout: 3x2`, `Layout 3x2 stacked` ,  `Layout: 1x3` 
- Create an update hook that moves blocks in sections using this class into two three-column layout(s)
  - Update hook for all sites using the `Layout: 3x2` and `Layout 3x2 stacked` 
    - The update hook should match the layout settings on https://sandbox.prod.drupal.uiowa.edu/stat-grid
      - Use the grid on admissions.uiowa.edu, [uiowa.](https://uiowa.edu/), https://uiowa.edu/about-iowa, https://writinguniversity.org/, https://ctgl.uiowa.edu for testing. 
  - No update hook for `Layout: 1x3` 

Related issues:

- https://github.com/uiowa/uiowa/issues/4317
- https://github.com/uiowa/uiowa/issues/3700

**Comment by bspeare** [comment]
I have updated the proposed solution to suggest that we remove these grid options and build an update hook for layouts using these grid options into our three-column layout.

**Comment by bspeare** [comment]
Example grid use on https://sitesflash-program.prod.drupal.uiowa.edu/people-0 to place images next to text areas. 

<img width="1227" alt="Image" src="https://github.com/user-attachments/assets/63aba461-595c-4ab0-8ea4-f493eb18f6fd" />

**Comment by bspeare** [comment]
Added https://github.com/uiowa/uiowa/issues/8643 as something we can do until we can get to this work.

**Comment by briand44** [comment]
Grooming conversation... Ideally we deprecate these existing options so they are no longer available. Discussed removing it from default content. May need a communication around this, disable section with a message for web editor on front end and have them update to avoid a very complex update hook?

Discussed scoping to include an update hook and if it gets too big or complex we can revisit if we can deliver on a smaller slice first. May need a scheduled release.

**Comment by bspeare** [comment]
Added more examples to https://sandbox.prod.drupal.uiowa.edu/stat-grid-unsaved-changes and https://sandbox.prod.drupal.uiowa.edu/stat-grid-without-unsaved-changes. 

- Example of two three column layouts stacked to recreate the 3x2 stacked grid 
- Two column 50/50 layout with grid 3x2 stacked style 
- Two column 50/50 layout using 3x2 stacked to create people listing 
- Three column 50/25/25 layout using 3x2 stacked 
- One column grid using 3x2 style 
- One column grid using 3x2 stacked style 
- One column grid using 1x3 style 

The main update hook should be for the One column grid using 3x2 stacked style example. Once that is complete we might want to do one for Two column 50/50 layout using 3x2 stacked to create people listing or to a query to see how much that one is used.

**Comment by joewhitsitt** [comment]
i can start looking at this after my 1:1 at 10. willing to pair

may have questions though

**Comment by joewhitsitt** [comment]
>The main update hook should be for the One column grid using 3x2 stacked style example

okay. so...

- Start off targeting `One column grid using 3x2 stacked style` `layout_onecol` `section_layout_grid_3x2_stacked`
- Move that over to multiple layout_threecol_section sections (groups of three) maintaining all of the existing section styles except for the grid style choice. Insert these right after original.
- Delete the original section once empty.
- Account for revisions/tempstore.

Do I have that right so far?

**Comment by joewhitsitt** [comment]
ack! hit a snag. right now I am just processing each section that matches the criteria (single delta, revision id, etc.), but then creating more than one section to replace it (two + stacked three-col), the delta+1, becomes a factor if another existing section has that delta or closer.

**Comment by joewhitsitt** [comment]
i am working on keeping track of the original delta, deleting the original, offsetting the higher deltas and inserting the new ones.  i am now worried that based on all of the db manipulation lb tempstore could be a potential fun hater.

**Comment by joewhitsitt** [comment]
it worked until it didn't. the db manipulation doesn't properly track multiple on a single node. I think it needs to be grouped and processed together if they share an entity_id+revision_id.

**Comment by joewhitsitt** [comment]
@bspeare How important is that "Make sure `remove_default_top_padding` is added for the first section only."?

Right now, I have it sending all existing styles over (background color, container, etc.), removing the grid style and making sure those padding and margin styles are included. I can do that all that outside the helper, but for the top padding one, that relies on knowing which section is being created which is very much within the helper function currently.

Would the visual difference cause a revolt if it was left in/removed if it existed?

**Comment by bspeare** [comment]
@joewhitsitt let's ignore that for now and we can see how it looks in testing.

**Comment by joewhitsitt** [comment]
Ok. Sounds good.

**Comment by joewhitsitt** [comment]
waiting to get feedback at EoD but i am very tempted to empty "tempstore" for nodes touched instead of trying to update them as well.

https://drupal.stackexchange.com/a/276028

**Comment by joewhitsitt** [comment]
@bspeare for `Two column 50/50 layout using 3x2 stacked` is there a replacement or is it just to audit so we can provide case-by-case recommendations?

**Comment by bspeare** [comment]
@joewhitsitt I added a replacement example at https://sandbox.prod.drupal.uiowa.edu/stat-grid-without-unsaved-changes#replacement.  Or we could run an audit to see how many sites are doing what is happening at https://github.com/uiowa/uiowa/issues/5351#issuecomment-2599141793.

**Comment by joewhitsitt** [comment]
i think i can process the two-columns but I need to adjust how the grouping is handled. i assume i need to group by region first then based on the number of components send it to a three or four column layout.

**Comment by joewhitsitt** [comment]
Question from standup:

Should we convert 1x3 to anything? For any remaining grid classes in empty sections or sections we aren't targeting (https://sandbox.prod.drupal.uiowa.edu/cta-4-col) should there just be a clean up update hook before the config is removed?

**Comment by bspeare** [comment]
@joewhitsitt Since the `1x3` is no longer working, I think we should remove the class and leave the blocks.

**Comment by joewhitsitt** [comment]
Had trouble with default content. I had updated the patch and the module. Not sure if that did anything but I was able to export stuff other than node/1. I reverted the [last change](https://github.com/uiowa/uiowa/pull/8484/files#diff-16abb31ec387fa3d78171e235c7f96dc51514254bbfc355abb7a54db08f18cce) to node/1 and did a site install before running through the process again. I made sure to apply the changes again to node/1 and was able to export again.

The export now includes a bunch of metatag stuff that we do not want because of the title, canonical, short link info. Removed that. Changes all the media aliases back (order of import issue).

Fixed some of the images so that it doesn't install with appended underscore stuff.

**Comment by joewhitsitt** [comment]
appendComponent or insertComponent is not respecting the previous weight of the component and so it all is added as weight = 0 which then affects how the blocks are called into the render.

things like setWeight and setComponent are protected.

Update: I can't seem to modify the component without deleting it and recreating it after it has been moved to the new section.

**Comment by joewhitsitt** [comment]
`./vendor/bin/blt ume "sqlq \"SELECT entity_id from node__layout_builder__layout WHERE layout_builder__layout_section LIKE '%section_layout_grid_3x2%';\""`

<details>
<summary>uiowa09</summary>

```
accessibility.uiowa.edu...
31

cfhsrcore.lab.uiowa.edu...
1

dalton.lab.uiowa.edu...
1

digital.studio.uiowa.edu...
1

ech2o.lab.uiowa.edu...
31
31
81
81

flash-program.sites.uiowa.edu...
1
51
51
51
51

harper.lab.uiowa.edu...
1

johannesledolter.studio.uiowa.edu...
1

krui.org.uiowa.edu...
1

mulfaul.lab.uiowa.edu...
1

muscog.lab.uiowa.edu...
1

petrin.lab.uiowa.edu...
1

thegammanupes1914.org.uiowa.edu...
1
```

</details>

<details>
<summary>uiowa08</summary>

```
blog.lib.uiowa.edu...
1

cat.lab.uiowa.edu...
1

cyndari.lab.uiowa.edu...
1
46

iowareview.org...
1

m-ashley-spies.lab.uiowa.edu...
1

oliviahubbardportfolio.sites.uiowa.edu...
1

pawh.lab.uiowa.edu...
1

prssa.org.uiowa.edu...
1

ukchoralcomposers.lib.uiowa.edu...
1

```

</details>

<details>
<summary>uiowa07</summary>

```
aboualaiwa.lab.uiowa.edu...
1

bluereview.sites.uiowa.edu...
1
211
216
221
231
236

ccsc.org.uiowa.edu...
1

celp.org.uiowa.edu...
1

cgrer.uiowa.edu...
1

colorguard.org.uiowa.edu...
1

cssi.research.uiowa.edu...
1

ctsdmc.uiowa.edu...
1

ebsl.lab.uiowa.edu...
31
36

econsent.lab.uiowa.edu...
1

ehs.research.uiowa.edu...
1

greenreview.sites.uiowa.edu...
181
186
191
196
206
211
216
221

hawkeyesba.sites.uiowa.edu...
1

hcha.org.uiowa.edu...
1

hhpeim-oc.org.uiowa.edu...
1

ieee.org.uiowa.edu...
1

iowaformularacing.org.uiowa.edu...
1

iowatc.org.uiowa.edu...
1

isc.research.uiowa.edu...
1

lcn.lab.uiowa.edu...
1

living.lab.uiowa.edu...
1

mclendon.lab.uiowa.edu...
1

mensicehockeyclub.org.uiowa.edu...
1

miller.lab.uiowa.edu...
1

muhs.org.uiowa.edu...
1

muhsstudent.org.uiowa.edu...
1

onlinepublishing.course.uiowa.edu...
1

orangereview.sites.uiowa.edu...
186
191
206

pedley.lab.uiowa.edu...
101
106

pezzulo.lab.uiowa.edu...
1

pgn.org.uiowa.edu...
1

psychotherapy-research.lab.uiowa.edu...
1

rai-me.sites.uiowa.edu...
1

redreview.sites.uiowa.edu...
1

report.hancher.uiowa.edu...
46
46
46
46
46

santillan.lab.uiowa.edu...
1

shariftabrizi.lab.uiowa.edu...
1

sparta.lab.uiowa.edu...
1

tivanskigroup.lab.uiowa.edu...
1

venues.uiowa.edu...
1

violetreview.sites.uiowa.edu...
1

whtr.lab.uiowa.edu...
1

worklife.uiowa.edu...
1

yellowreview.sites.uiowa.edu...
236
241
251
256
261

yu.lab.uiowa.edu...
1

```

</details>

<details>
<summary>uiowa06</summary>

```

anesthesia.medicine.uiowa.edu...
1

appil.medicine.uiowa.edu...
1

biomed.medicine.uiowa.edu...
1

bmb.medicine.uiowa.edu...
726

cpss.uihealthcare.org...
1

demo.medicine.uiowa.edu...
1
151
281

dermatology.medicine.uiowa.edu...
1
126
131

gme.uihealthcare.org...
1

humangenetics.medicine.uiowa.edu...
1
631

icts.uiowa.edu...
1

immunologycenter.medicine.uiowa.edu...
201
201
201
201
211
211
211
211
216
216
216
216
221
221
221
221
221
221
221
221
226
226
226
231
231
236
236
236
241
241
241
241
246
246
246
251
251
261

inbc.medicine.uiowa.edu...
1

inflammation.medicine.uiowa.edu...
1
46
56
91
91
91
156
156
156
161
161
161
161
161
171

iowaneuroscience.medicine.uiowa.edu...
1

md.medicine.uiowa.edu...
1

mmed.medicine.uiowa.edu...
1

mri.medicine.uiowa.edu...
41

nbs.medicine.uiowa.edu...
1

neurology.medicine.uiowa.edu...
1
476
531
546
616
621
651
656

neurosurgery.medicine.uiowa.edu...
1

obesity.medicine.uiowa.edu...
1

otolaryngology.medicine.uiowa.edu...
1

pain.medicine.uiowa.edu...
1

pathology.medicine.uiowa.edu...
1
1
871
1301
1301

pbi.medicine.uiowa.edu...
1

pediatrics.medicine.uiowa.edu...
1

research.medicine.uiowa.edu...
1
231

respiratorycareseminar.uihealthcare.org...
1

surgery.medicine.uiowa.edu...
1

uidl.medicine.uiowa.edu...
1

uiphysicians.medicine.uiowa.edu...
1

```

</details>

<details>
<summary>uiowa05</summary>

```
baker.lab.uiowa.edu...
31

bjj.org.uiowa.edu...
1

bnelson.lab.uiowa.edu...
1
96

catering.uiowa.edu...
1
151
166

cytogenetics.lab.uiowa.edu...
1
31

dining.uiowa.edu...
1
446

dpriester.sites.uiowa.edu...
1

epsm.lab.uiowa.edu...
1

exchanges.uiowa.edu...
1

ferri.lab.uiowa.edu...
31

geerling.lab.uiowa.edu...
1

glykys.lab.uiowa.edu...
1

hallieroach.sites.uiowa.edu...
1

hccc-oinc.sites.uiowa.edu...
1

hydrogeology.lab.uiowa.edu...
1

i3hsa.org.uiowa.edu...
1

iise.org.uiowa.edu...
1

infant-communication-development.lab.uiowa.edu...
1

kerns.lab.uiowa.edu...
1

learningspaces.org.uiowa.edu...
1
31

lewisstevens.lab.uiowa.edu...
1

mind.lab.uiowa.edu...
1

mocktrial.org.uiowa.edu...
1
31
51
61

nickljockschat.lab.uiowa.edu...
1

ophthalmologyinterestgroup.org.uiowa.edu...
1

pcit.lab.uiowa.edu...
1

peng.lab.uiowa.edu...
51

petersen.lab.uiowa.edu...
1

produnit.clas.uiowa.edu...
1

rkinser.sites.uiowa.edu...
1

trips.lab.uiowa.edu...
96

uiowachoirauditions.org.uiowa.edu...
1

uisrmsymposium.sites.uiowa.edu...
1

wellbeing.uiowa.edu...
381

xryouthethics.org.uiowa.edu...
1

young.lab.uiowa.edu...
1

```

</details>

<details>
<summary>uiowa04</summary>

```
anthonyfischer.lab.uiowa.edu...
1

auditorytrainingindex.sites.uiowa.edu...
1


beattie.lab.uiowa.edu...
36
36
36
36
51

bell.lab.uiowa.edu...
1

cbb.research.uiowa.edu...
1

cdd.center.uiowa.edu...
1

climate-change-lung-health.sites.uiowa.edu...
1

ctgl.uiowa.edu...
346
366

dholterhaus.sites.uiowa.edu...
1

dsri.uiowa.edu...
216
221

emfisis.sites.uiowa.edu...
1

engelhardt.lab.uiowa.edu...
31

epicsupport.sites.uiowa.edu...
36

hccc-summer-undergrad.sites.uiowa.edu...
1

icarepostbac.sites.uiowa.edu...
1

idream.sites.uiowa.edu...
1

intranet.writersworkshop.uiowa.edu...
76
906
906

muacc.conference.uiowa.edu...
1

narayanan.lab.uiowa.edu...
31

niciu.lab.uiowa.edu...
1

nola.lab.uiowa.edu...
1

our.research.uiowa.edu...
1791

pih-engage.org.uiowa.edu...
1

policy.uiowa.edu...
1

politicalscience.uiowa.edu...
6

researchpark.uiowa.edu...
1
66

saah-design.studio.uiowa.edu...
1

safety.uiowa.edu...
101

sebag.lab.uiowa.edu...
1

sloancenter.uiowa.edu...
1

spark.sites.uiowa.edu...
1

thorne.lab.uiowa.edu...
1

thornell.lab.uiowa.edu...
1

turkish.org.uiowa.edu...
1

victorianfboos.studio.uiowa.edu...
1

writersworkshop.uiowa.edu...
1
316
316
316
316

```

</details>

<details>
<summary>uiowa03</summary>

```
anstig.org.uiowa.edu...
1

archaeology.uiowa.edu...
1
276
276
276
276
471
471
471
471
521
521
521
521

boxing.org.uiowa.edu...
1

carter.lab.uiowa.edu...
1

chatterjee.lab.uiowa.edu...
31

chen.lab.uiowa.edu...
1

cmhw.lab.uiowa.edu...
1

collections.sites.uiowa.edu...
1

cssa.org.uiowa.edu...
1

ethanborchard.org.uiowa.edu...
1

glefevre.lab.uiowa.edu...
1
31

hancher.uiowa.edu...
1

hansen.lab.uiowa.edu...
1

innovation.uiowa.edu...
31
31
31
31
31
36
36
56

intranet.pharmacy.uiowa.edu...

iowabrainbank.lab.uiowa.edu...
1

iowaedge.sites.uiowa.edu...
1

iowahumanrights.org...
1

iowamobileclinic.org.uiowa.edu...
1

irrc.education.uiowa.edu...
311
311

kenny.lab.uiowa.edu...
1

languageacrossthelifespan.lab.uiowa.edu...
56

magidcenter.uiowa.edu...
56

mu.lab.uiowa.edu...
56
291

nasa.org.uiowa.edu...
1

neurovascular.lab.uiowa.edu...
1

pmril.lab.uiowa.edu...
1

prehealth.advisingcenter.uiowa.edu...
1

pwp.org.uiowa.edu...
31
81
86
91
96
271
746

race-and-comics.sites.uiowa.edu...
1

rad.lab.uiowa.edu...
36

rodgers.lab.uiowa.edu...
1

roghair.lab.uiowa.edu...
31

rutkowski.lab.uiowa.edu...
1

salem.lab.uiowa.edu...
1

sheffield.lab.uiowa.edu...
1

sinfonia.org.uiowa.edu...
1

sipe.org.uiowa.edu...
1

uisqrp.music.uiowa.edu...
1

uitransalliance.org.uiowa.edu...
1

vapap.org.uiowa.edu...
1

wallrath.lab.uiowa.edu...
1

womensrugby.org.uiowa.edu...
76

```

</details>

<details>
<summary>uiowa02</summary>

```
admissions.uiowa.edu...
1
1
26
31
4491

arsc.lab.uiowa.edu...
21

bhattacherjee.lab.uiowa.edu...
51

boes.lab.uiowa.edu...
26

csomaycenter.uiowa.edu...
881

cuhal.lab.uiowa.edu...
31
31
36

currents.iihr.uiowa.edu...
1
1
81
81

education.uiowa.edu...
1
6
6
86
436

engage.uiowa.edu...
1

executive.jobs.uiowa.edu...
1

folland.lab.uiowa.edu...
6

glefevre.iihr.uiowa.edu...
1

grad.admissions.uiowa.edu...
26
166
1696

iddrc.uiowa.edu...
1
41
81
81
496
506
506

iisc.uiowa.edu...
81

iowajpec.org...
2046

jtucker.lab.uiowa.edu...
1
1

kawai.lab.uiowa.edu...
21
51

ohsc.lab.uiowa.edu...
1

physicsresearch.sites.uiowa.edu...
31

provost.uiowa.edu...
301

recserv.uiowa.edu...
1

sandbox.uiowa.edu...
851
856
1596
1596
1596
1596
1596
1596
1596
2346
2346
2486
2746
2776
2776
2776
2776
2776
2776
2776

scholardev.sites.uiowa.edu...
51
71
81
111

sportsmedicine.athletics.uiowa.edu...
1

stanleymuseum.uiowa.edu...
1311
1311
1311
1311
3431

strategicplan.uiowa.edu...
831
841

studentsuccess.uiowa.edu...
21

tippie.uiowa.edu...
846
1156
1356
1591
1601
1851
1851
1896
1926
1926
1936
1946
1981
2041
2041
2136
2226
2231
2236
2241
2246
2251
4716
5151
8011
8056
8056
8171

uimsa.org.uiowa.edu...
1
1
1

uiprehealth.conference.uiowa.edu...
1
1

webcommunity.sites.uiowa.edu...
466

```

</details>

<details>
<summary>uiowa01</summary>

```
accessibility101.course.uiowa.edu...
1
31

advisingcenter.uiowa.edu...
276
381
386
911
911
971

asian-slavic.uiowa.edu...
646

bam.sites.uiowa.edu...
1

carterradiochem.lab.uiowa.edu...
1

clcl.uiowa.edu...
906

german.uiowa.edu...
556

grad.uiowa.edu...
201

gurnett.physics.uiowa.edu...
1

hipaa.uiowa.edu...
26

itpmo.uiowa.edu...
1

iwp.uiowa.edu...
256

jacobsoninstitute.org...
1
101
366

mgnait.lab.uiowa.edu...
101

ncec.sites.uiowa.edu...
56

neuroimaging.uiowa.edu...
341
376
386

obermann.uiowa.edu...
376
381
386
391
396
401
406
411
416
421
426
1786

qa.physics.uiowa.edu...
31

quench.org.uiowa.edu...
26

spanish-portuguese.uiowa.edu...
936

uiowa.edu...
1
2
2
56
161

writinguniversity.org...
2
91
106
116
171
4111
4331

```

</details>

<details>
<summary>uiowa</summary>

```
charlie.uiowa.edu...
1261
1681
1681
1681

classrooms.uiowa.edu...
166
361

clp.law.uiowa.edu...
1

coi.research.uiowa.edu...
1

default...
1

dentistry.uiowa.edu...
6
486
871
876
881
891
891
1312
3446

dsp.research.uiowa.edu...
1

engineering.uiowa.edu...
76
5686
6071
6221
6266
6286
7226
8306
9006
9126
9126
9126
9126
9126
9126
9126
9126
9126
9736
9736
9846
9876
9876
9876
9876
9876
9876
9876
9876
9876
9876
9876
9876
9876
9876
9876
10966
11076
11146
11146
11146
11146
11146
11146
11146

fmb.fo.uiowa.edu...
1

heli.law.uiowa.edu...
96
196
321
326

help.maui.uiowa.edu...
416
726

hso.research.uiowa.edu...
1

international.uiowa.edu...
1
111
831
1776
13696

iti.uiowa.edu...
1
2676
2676
2676

iwwerp.engineering.uiowa.edu...
1

jgrj.law.uiowa.edu...
1

karandikar.lab.uiowa.edu...
31

pchs.lhpdc.law.uiowa.edu...
1

research.uiowa.edu...
1
266
671
696
3416
4196
4226

sitenow.uiowa.edu...
1

tedabel.lab.uiowa.edu...
106

training.itaccessibility.uiowa.edu...
186

uirf.research.uiowa.edu...
151

uiservicecenter.uiowa.edu...
1
31

```

</details>

**Comment by bspeare** [comment]
@joewhitsitt I think that's a manageable list to spot-check after deployment.  Some pages it will completely help like https://iddrc.uiowa.edu/research-cores/clinical-translational-core/patient-recruitment-clinical-evaluation-division and https://obermann.uiowa.edu/articles-about-hpl-book-series.  And some pages will need to be adjusted like https://immunologycenter.medicine.uiowa.edu/neuro-and-ophthalmic-immunology.

**Comment by joewhitsitt** [comment]
Here is a `dsa` compatible list of non-node/1:

```
multisites:
  - accessibility.uiowa.edu
  - ech2o.lab.uiowa.edu
  - flash-program.sites.uiowa.edu
  - cyndari.lab.uiowa.edu
  - bluereview.sites.uiowa.edu
  - ebsl.lab.uiowa.edu
  - greenreview.sites.uiowa.edu
  - orangereview.sites.uiowa.edu
  - pedley.lab.uiowa.edu
  - report.hancher.uiowa.edu
  - yellowreview.sites.uiowa.edu
  - bmb.medicine.uiowa.edu
  - demo.medicine.uiowa.edu
  - dermatology.medicine.uiowa.edu
  - humangenetics.medicine.uiowa.edu
  - immunologycenter.medicine.uiowa.edu
  - inflammation.medicine.uiowa.edu
  - mri.medicine.uiowa.edu
  - neurology.medicine.uiowa.edu
  - pathology.medicine.uiowa.edu
  - research.medicine.uiowa.edu
  - baker.lab.uiowa.edu
  - bnelson.lab.uiowa.edu
  - catering.uiowa.edu
  - cytogenetics.lab.uiowa.edu
  - dining.uiowa.edu
  - ferri.lab.uiowa.edu
  - learningspaces.org.uiowa.edu
  - mocktrial.org.uiowa.edu
  - peng.lab.uiowa.edu
  - trips.lab.uiowa.edu
  - wellbeing.uiowa.edu
  - beattie.lab.uiowa.edu
  - ctgl.uiowa.edu
  - dsri.uiowa.edu
  - engelhardt.lab.uiowa.edu
  - epicsupport.sites.uiowa.edu
  - intranet.writersworkshop.uiowa.edu
  - our.research.uiowa.edu
  - politicalscience.uiowa.edu
  - researchpark.uiowa.edu
  - safety.uiowa.edu
  - writersworkshop.uiowa.edu
  - archaeology.uiowa.edu
  - chatterjee.lab.uiowa.edu
  - glefevre.lab.uiowa.edu
  - innovation.uiowa.edu
  - irrc.education.uiowa.edu
  - languageacrossthelifespan.lab.uiowa.edu
  - magidcenter.uiowa.edu
  - mu.lab.uiowa.edu
  - pwp.org.uiowa.edu
  - rad.lab.uiowa.edu
  - roghair.lab.uiowa.edu
  - womensrugby.org.uiowa.edu
  - admissions.uiowa.edu
  - arsc.lab.uiowa.edu
  - bhattacherjee.lab.uiowa.edu
  - boes.lab.uiowa.edu
  - csomaycenter.uiowa.edu
  - cuhal.lab.uiowa.edu
  - currents.iihr.uiowa.edu
  - education.uiowa.edu
  - folland.lab.uiowa.edu
  - grad.admissions.uiowa.edu
  - iddrc.uiowa.edu
  - iisc.uiowa.edu
  - iowajpec.org
  - kawai.lab.uiowa.edu
  - physicsresearch.sites.uiowa.edu
  - provost.uiowa.edu
  - sandbox.uiowa.edu
  - scholardev.sites.uiowa.edu
  - stanleymuseum.uiowa.edu
  - strategicplan.uiowa.edu
  - studentsuccess.uiowa.edu
  - tippie.uiowa.edu
  - webcommunity.sites.uiowa.edu
  - accessibility101.course.uiowa.edu
  - advisingcenter.uiowa.edu
  - asian-slavic.uiowa.edu
  - clcl.uiowa.edu
  - german.uiowa.edu
  - grad.uiowa.edu
  - hipaa.uiowa.edu
  - iwp.uiowa.edu
  - jacobsoninstitute.org
  - mgnait.lab.uiowa.edu
  - ncec.sites.uiowa.edu
  - neuroimaging.uiowa.edu
  - obermann.uiowa.edu
  - qa.physics.uiowa.edu
  - quench.org.uiowa.edu
  - spanish-portuguese.uiowa.edu
  - uiowa.edu
  - writinguniversity.org
  - charlie.uiowa.edu
  - classrooms.uiowa.edu
  - dentistry.uiowa.edu
  - engineering.uiowa.edu
  - heli.law.uiowa.edu
  - help.maui.uiowa.edu
  - international.uiowa.edu
  - iti.uiowa.edu
  - iwwerp.engineering.uiowa.edu
  - karandikar.lab.uiowa.edu
  - research.uiowa.edu
  - tedabel.lab.uiowa.edu
  - training.itaccessibility.uiowa.edu
  - uirf.research.uiowa.edu
  - uiservicecenter.uiowa.edu
 
```

Should we create a follow-up task to spot-check sites post-deployment? Also, this seems to largely be a padding before/after type issue am I correct? Is the juice worth the squeeze here? Do we instead assume the worst and make sure extra padding is added to top and bottom sections? Is that the lesser of two evils?

**Comment by bspeare** [comment]
I'd like to spot check them.  Yes, it is mostly just a padding before/after issue.   But spot checking will also help us fix layouts that were previously broken.

**Comment by joewhitsitt** [comment]
Created a separate follow-up issue: https://github.com/uiowa/uiowa/issues/8708
